src/transformers/models/seggpt/configuration_seggpt.py

```diff
@@ -0,0 +1,166 @@
+            The pretrained size of the absolute position embeddings.
+        decoder_hidden_size (`int`, *optional*, defaults to 64):
+            Hidden size for decoder.
+        use_rel_pos (`bool`, *optional*, defaults to `True`):
+            Whether to use relative position encoding in the Attention
+        merge_index (`int`, *optional*, defaults to 2):
+            The index of the encoder layer to merge the embeddings.
+        out_indicies (`List[int]`, *optional*, defaults to `[5, 11, 17, 23]`):
+            The indices of the encoder layers which we store as features for the decoder.
+        beta (`float`, *optional*, defaults to 0.01):
+            Regularization factor for SegGptLoss (smooth-l1 loss).
+
+    Example:
+
+    ```python
+    >>> from transformers import SegGptConfig, SegGptModel
+
+    >>> # Initializing a SegGpt seggpt-base-patch16-224 style configuration
+    >>> configuration = SegGptConfig()
+
+    >>> # Initializing a model (with random weights) from the seggpt-base-patch16-224 style configuration
+    >>> model = SegGptModel(configuration)
+
+    >>> # Accessing the model configuration
+    >>> configuration = model.config
+    ```"""
+
+    model_type = "seggpt"
+
+    def __init__(
+        self,
+        hidden_size=1024,
+        num_hidden_layers=24,
+        num_attention_heads=16,
+        hidden_act="gelu",
+        hidden_dropout_prob=0.0,
+        initializer_range=0.02,
+        layer_norm_eps=1e-6,
+        image_size=[896, 448],
+        patch_size=16,
+        num_channels=3,
+        qkv_bias=True,
+        mlp_ratio=4.0,
+        drop_path_rate=0.1,
+        pretrain_img_size=224,
+        decoder_hidden_size=64,
+        use_rel_pos=True,
+        merge_index=2,
+        out_indicies=[5, 11, 17, 23],
+        beta=0.01,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+
+        self.hidden_size = hidden_size
+        self.num_hidden_layers = num_hidden_layers
+        self.num_attention_heads = num_attention_heads
+        self.hidden_act = hidden_act
+        self.hidden_dropout_prob = hidden_dropout_prob
+        self.initializer_range = initializer_range
+        self.layer_norm_eps = layer_norm_eps
+        self.image_size = image_size
+        self.patch_size = patch_size
+        self.num_channels = num_channels
+        self.qkv_bias = qkv_bias
+        self.mlp_ratio = mlp_ratio
+        self.drop_path_rate = drop_path_rate
+        self.pretrain_img_size = pretrain_img_size
+        self.decoder_hidden_size = decoder_hidden_size
+        self.use_rel_pos = use_rel_pos
+        if merge_index > min(out_indicies):
+            raise ValueError(
+                f"Merge index must be less than the minimum encoder output index, but got {merge_index=} and {out_indicies=}"
+            )
+        self.merge_index = merge_index
+        self.out_indicies = out_indicies
+        self.beta = beta
+
+
+class SegGptOnnxConfig(OnnxConfig):
```

Comment:
Can be removed

Reply:
https://github.com/huggingface/transformers/pull/27735/commits/52de2a76ffba72604469831d102d62621e1b8cc9